core/src/main/java/de/danoeh/antennapod/core/util/playback/PlaybackController.java

```diff
@@ -750,6 +750,7 @@ public void notifyVideoSurfaceAbandoned() {
     public void reinitServiceIfPaused() {
         if (playbackService != null
                 && playbackService.isStreaming()
+                && !PlaybackService.isCasting()
```

Comment:
I believe the bandwidth argument isn't valid anymore once casting, considering the communication between devices just to cycle through the states of the reloading phase.